ingestion/src/metadata/ingestion/source/database/postgres_lineage.py

```diff
@@ -33,7 +33,7 @@ class PostgresLineageSource(PostgresQueryParserSource, LineageSource):
     filters = """
                 AND (
                     s.query ILIKE '%%create table %% as select %%'
```

Comment:
same applies for postgress as well for queries like `create table as (select ....) `